src/forked/marked/bower.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
The `bower.json`/`component.json` and `package.json` should probably be deleted as well, as this won't be used as a separate module. The `devDependencies` appear to be only used by the Gulpfile.js which I think should be deleted as well.